spec/board_spec.rb

```diff
@@ -42,4 +42,11 @@
     end
   end
 
+  context 'when a single token occupies all given positions on the board' do
```

Comment:
Perhaps it'll be easy to group tests that belong to the same group by using another describe for the method that you're testing:
```ruby
describe Board do
  describe '#found_winner_at' do
```

Reply:
This begs a question of when describe should be used. I always thought it could only be used when describe classes. IS that not entirely true?

Reply:
Good question! You can definitely use describe within a describe. I think in this case it'll help organize the test since we're testing multiple method inside `Board`